src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.1.0.4395")]
+[assembly: AssemblyVersion("2.1.0.6957")]
```

Comment:
you should usually not commit this file.

Reply:
Whoops, reverting this